### DISCUSSION TITLE
invalidateQueries를 잘 하기 위한 queryKey id 타입 통일 (X)

### DISCUSSION THREAD

**Discussion by un0211** [Part3-Team15/taskify#146] [General]
queryKey에서 id 타입을 통일해두어야 나중에 변경사항이 생겨서 invalidateQueries를 호출할 때 제대로 동작합니다.
fetch 진행할 때 queryKey에 넣을 id 타입을 논의해보면 좋을 것 같아요!
(지금은 일일이 fetch 코드를 확인하고, 그에 맞는 타입을 invalidate 해주어야합니다.

**Comment by wjsdncl** [comment]
`useFetchData<InvitationsResponse>(['invitations'], () => getInvitationsList());`
fetch 할때 사용하는 'invitations'를 id로 수정하자는 말씀이신건가요? 아니면 id를 추가하는 건가요? 🤔

**Reply by un0211** [reply]
['dashboards', id] 면 id가 string 이냐 number 냐에 따라 달라집니다!
그래도 dashboard는 router로 가져와서 대부분 string이긴 하더라구요
